Makefile

```
@@ -17,11 +17,13 @@ coverage:
 
 .PHONY: doc
 doc:
+	python3 -c "import PIL" > /dev/null 2>&1 || python3 -m pip install .
+	python3 -c "import olefile" > /dev/null 2>&1 || python3 -m pip install olefile
```

We're building Pillow anyway, shall we just use the docs extra?

```suggestion
	python3 -c "import PIL" > /dev/null 2>&1 || python3 -m pip install ".[docs]"
```

---

If Pillow was already installed, but olefile wasn't, your version wouldn't install it. Unless you'd like to add olefile to the docs/Makefile install command?

---

Yes, makes more sense to move it to docs/Makefile.